src/main/java/ch/wisv/areafiftylan/service/TeamServiceImpl.java

```diff
@@ -108,7 +108,12 @@ public TeamInviteToken inviteMember(Long teamId, String username) {
         User user = userService.getUserByUsername(username).orElseThrow(() -> new UsernameNotFoundException(username));
         Team team = getTeamById(teamId);
 
-        if (!team.getMembers().contains(user)) {
+        // Check if the member isn't already invited
+        List<TeamInviteToken> currentTeamInvites = teamInviteTokenRepository.findByUserUsername(username).stream()
+                .filter(token -> token.getTeam().equals(team)).collect(Collectors.toList());
```

Comment:
A bit more optimized would be using `Stream::findFirst`. For large sets, it stops immediately when it finds the item, instead of iterating over the whole set. This solution of course works too, just an optimization.